identity/src/main/java/com/stripe/android/identity/navigation/FragmentKtx.kt

```diff
@@ -0,0 +1,19 @@
+package com.stripe.android.identity.navigation
+
+import androidx.fragment.app.Fragment
+import com.stripe.android.identity.IdentityActivity
+
+/**
+ * Gets the hosting [IdentityActivity] and invokes [IdentityActivity.ensureCameraPermission].
+ */
+fun Fragment.ensureCameraPermissionFromIdentityActivity(
```

Comment:
Since all fragments are hosted by the single activity, the cast is guaranteed to success